code/src/components/Task.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+import moment from 'moment';
+
+import { tasks } from 'reducers/tasks';
+
+import { CustomCheckbox } from './CustomCheckbox';
+
+import { TaskText, Emoji, SmallText } from 'styles/GlobalStyles';
+
+export const Task = ({ task }) => {
+  const dispatch = useDispatch();
+
+  const handleCheckboxClick = () => {
+    dispatch(tasks.actions.markCompleted(task.id))
+  };
+
+  const handleRemoveButtonClick = () => {
+    dispatch(tasks.actions.removeTask(task.id))
+  };
+
+  return (
+    <List>
+      <Wrapper>
+        <CustomCheckbox 
+          isChecked={task.complete} 
+          onChangeHandler={handleCheckboxClick}
+        />
+        <TaskText line={task.complete ? 'line-through' : null}>{task.category} {task.text}</TaskText>
+      </Wrapper>
+      <Wrapper>
+      <SmallText>Due: {moment(task.dueDate).format('MMM Do YYYY')}</SmallText>
```

Comment:
Either force the user to pick a date or show something else (or nothing) if the user has not picked a date :)

Reply:
Smart! Thanks for the tip 🤩